build/scripts/contiv-vol-run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+set -e
+
+# cleanup the older container
+docker rm -f "$1" &>/dev/null || :
+
+## test for shared mount capability
+if ! grep "MountFlags" /lib/systemd/system/docker.service | grep shared &>/dev/null
+then
+  echo 1>&2 "Docker cannot run volplugin in its current state."
+  echo 1>&2 "You must change docker's systemd MountFlags to equal 'shared'"
```

Comment:
> Is there some reason this can't be done in hyperkube?

Yeah, I think hyperkube requires some host volumes (like /var/lib/kubelet) to be mounted in shared mode, which fails if Docker is started with `MountFLags=slave`. I think moer details are there in the issue thread I shared above.